Integrates bitstring status list

kezike opened the pull request:
This PR integrates the Bitstring Status List.

dmitrizagidulin commented:
@kezike thanks! Reviewing now. (Can you fix the lint/indentation errors meanwhile?)

dmitrizagidulin commented on app/components/VerificationStatusCard/VerificationStatusCard.tsx:
```diff
@@ -80,7 +81,18 @@ export default function VerificationStatusCard({ credential, verifyPayload }: Ve
+            details[LogId.RevocationStatus]
+          }
+        />
+        <StatusItem
```
Let's display this item conditionally -- only when there's a `credentialStatus` section. (Ideally, we'd only display it if the bitstring status credential had suspension type lists)

kezike replied:
@dmitrizagidulin I have changed this to display this section only if suspension is being tracked in the credential. For consistency, I am also only displaying revocation status if it is being tracked in the credential. Let me know what you think.

dmitrizagidulin replied:
Thanks!

dmitrizagidulin reviewed (approved):
👍 looks good, pending:
* Fix linting errors on the Files Changed tab
* Display the 'suspended' line conditionally.

kezike commented:
@dmitrizagidulin I have addressed all of your concerns and I also made additional modifications regarding validity period, so that the app can support both versions (`issuanceDate` vs. `validFrom` and `expirationDate` vs. `validUntil`).
